### PR TITLE
Added a note

### PR DESCRIPTION
**Pull request by joinimran**
ATP doesn't need any specific diagnostic level.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/4824

### CODE REVIEW

**Comment by jvsam** [comment]
Hello @JohanFreelancer9 and @nenonix, just waiting for the revision from @joinimran. But please do a copy review. Thank you.

File: windows/security/threat-protection/microsoft-defender-atp/minimum-requirements.md [hunk 1]
```diff
@@ -99,6 +99,10 @@ When you run the onboarding wizard for the first time, you must choose where you
 
 
 ### Diagnostic data settings
+
+> [!NOTE]
+> Microsoft Defender ATP doesn't require any specific diagnostic level as long as its enabled.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/minimum-requirements.md]:
```suggestion
> Microsoft Defender ATP doesn't require any specific diagnostic level as long as it's enabled.
```

**Comment by joinimran** [comment]
Suggestions applied.

**Comment by jvsam** [comment]
Hello @mjcaparas, this pull request is ready for your review and approval. The suggested change is based on your feedback as regards to the diagnostic level. Please approve and merge if there are no issues. Thank you.

**Comment by jvsam** [comment]
Thank you for approving and merging this PR @mjcaparas!

**Pull request merged**